xtask/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+edition = "2021"
```

Comment:
this should be marked as private to avoid accidentally trying to publish this crate